### ISSUE TITLE
Update notice is too small and unnoticeable

### ISSUE THREAD

**Issue by xt0rted** [desktop/desktop#1632]
### Description

The banner saying there's an update available is almost unnoticeable and very hard to interact with because of its size. This happened when 0.5.3 prompted me about 0.5.4 as well.

I'm running at 4k and 150% scaling which has caused issues in the past with Slack and a number of other Electron based apps.

The css for `#update-available.active` puts this at `30px` high, but it's not until I make it `90px` high that it looks correct.

**Default - 30px**
![image](https://cloud.githubusercontent.com/assets/831974/26221314/06a2d118-3be4-11e7-8da0-6f4b75ffa538.png)

**Modified - 90px**
![image](https://cloud.githubusercontent.com/assets/831974/26221453/8e2f7122-3be4-11e7-8526-299533ff6f81.png)

### Version

**GitHub Desktop version:** 0.5.3, 0.5.4
**OS version:** Win10 Creators Update (10.0.15063)

**Comment by xt0rted** [comment]
I didn't restart the app since being prompted and what I've noticed is the update banner is sized differently based on the file that's selected in the Changes list. The size in the original report is based on the first file being selected, if any other file in the list is selected then the banner is sized correctly.

I had left the css rule modified to `90px` and this is what the banner looks like when selecting the second file on the list.

![image](https://cloud.githubusercontent.com/assets/831974/26223029/69395134-3bea-11e7-8f0c-bd4cee2afae0.png)

Moving back to the first file changes the height back to that of the **Modified - 90px** image from the initial report.

Switching to the History list has similar results, but instead of it being just the first commit that has a different size it's every other commit. So the first commit is sized incorrectly, the second is sized correctly, the third is sized incorrect, etc. The change is size is even more apparent when the css rule is set to `90px` since is switches between ~`30px` and `90px`.

**Comment by shiftkey** [comment]
Looks like the gremlins I stumbled upon in https://github.com/desktop/desktop/issues/1330 are back. Going to leave this open as you've got more details around reproducing it.

**Comment by niik** [comment]
> The banner saying there's an update available is almost unnoticeable

The bug that you mentioned about the weird sizing of the auto update box is definitely something we need to look at but as for the update notification being small and "unnoticeable" that is definitely a conscious choice we've made, see https://github.com/desktop/desktop/issues/1136.

We've designed the update notification to be subtle and unobtrusive so that we don't distract users as they're working. It annoys me when an application (looking at you [Sketch](https://sketchapp.com/)) throws a dialog in my face about an update as I'm working. With the update bar you'll notice it when you notice it, it's not critical for us that users update right the second they see the bar.

> [...] very hard to interact with because of its size.

This, however, is certainly something we can work with. We should ensure that the hit area for the `what's new` and `restart now` buttons cover the full height of the update notification.
